### ISSUE TITLE
[Platform SRE] Review existing incident response documentation 

### ISSUE THREAD

**Issue by jhouse-solvd** [department-of-veterans-affairs/va.gov-team#36810]
## Description
There is existing incident response documentation. We need to understand what exists before we can update the documentation for accuracy. 

## Background/context 
- [Incident commander](https://vfs.atlassian.net/wiki/spaces/ECP/pages/1745977409/Incident+Commander)
- [Incident categorization](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/OnCall/incident-categorization.md)
- [Incident response playbook](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/Incident%20Response%20Playbook.md) 

## Technical notes
_Notes around work that is happening, if applicable (optional, please delete if unused)_

---
## Tasks
- [x] Review incident commander documentation 
- [x] Review incident categorization documentation 
- [x] Review incident response playbook 

## Definition of done
- Existing documentation has been reviewed 
- New issues have been created to track specific changes to documentation

**Comment by zachclarity** [comment]
@jhouse-solvd  Discuss current documentation and finalize tasks to update if needed - Meeting planned for Feb 15th Tues.  at 2:15 EST -- Zoom Meeting
https://us06web.zoom.us/j/82133289126?pwd=ampHUWJOWXRrdkVBZ0JkU0w5Y3IwQT09

**Comment by zachclarity** [comment]
@jhouse-solvd  Question:  Are all of the products in this [spreadsheet of VSP products](https://docs.google.com/spreadsheets/d/1Fn2lD419WE3sTZJtN2Ensrjqaz0jH3WvLaBtn812Wjo) part of the incident process ?   What would be excluded and how should we document that.

**Comment by zachclarity** [comment]
@jhouse-solvd  How can we add a column to the spreadsheet to add dashboard or monitor column for to check as part of the incident process ?  How much details should we include in the process ?  I would like a step by step flow-chart for the products and Datadog or other dashboard to make it easy to link in to info about current state.   I know we need to add monitors and event and started this outline of one enhancement.  https://vfs.atlassian.net/wiki/spaces/SRE/pages/2088927233/WIP+-+Incident+Response+Monitors+Dashboards+Enhancements

**Comment by jhouse-solvd** [comment]
Some questions that have come out of the incident commander document review: 

Will we need to update slack channel names?  
What is the process to update points of contact?     
What happens if PagerDuty, Internet, or other tools fail?  
Is there backup phone contact information? 
Is the OnCall README accurate and up-to-date? 

Fix this: 
"Incident commanders should follow the Basic On Call Procedure" ... that document is deprecated 

More questions: 
How do we get SRE team to receive IC emails or PagerDuty prompts? 
Is the Security Incident Response document up to date (SIR)? 
Where is the ATO Incident Response Plan that's supposed to be linked from the SIR? 
Are the contacts up to date in the team-sensitive Administrative README? 
Do Incident Response personnel need to have a PIV card? 
How do we clarify the handoff and assumption of responsibilities between oncall and incident commander?
Who are VSP DEPO leads and where is that documented? 
Do we still post a message in #general? 
Are there other channels? 
How do we streamline broad notification in DSVA Slack? 
Is "vsp-leadership" still an active group?  
Where are the call center points of contact documented? 
Is the HRCCMOHDSupervisors@va.gov email list still valid? 
What is the "Emergency Operating Procedures Incident Response Plan" and where is it documented? 
What is the "Incident Response Standard Operating Procedures - ATO" and where is it documented?

**Comment by jhouse-solvd** [comment]
Incident categorization questions: 

Are there too many categories? 
It seems severe and critical are quite similar? 
How do we define a timeline for varying levels of urgency? 
Are there SLAs that can guide how quickly or how urgent we should respond? 
Can we draw a line between issues that on-call personnel receive versus things that are classified as incidents? Or that require an incident commander?  
Are minor incidents actually incidents? Or are they just issues? 
Which environments does this apply to? 
For minor, normal, and major, the categorization does not currently require escalating to the incident commander? Do we want to rope in the IC role for those incidents? During business hours? During off hours? 
Is there an incident issue template? 
How does the IC get updated if a M/N/M incident occurs and they were not paged? 
How do we communicate M/N/M incidents between the on-call personnel and the IC?

**Comment by oseasmoran73** [comment]
Adding [triage](https://dsva.slack.com/archives/CTYQL39FE/p1650038597113689) thread

Given our recent incident, I gave some feedback on potential improvements. I think we can do one the following:
- Officially close out this ticket and spin up a new ticket with feedback given (see thread where me and @zachclarity are chatting)
- Resurface this from `review` back to `in progress`

I personally am for the former as it keeps a bit clean and we can always reference this one for additional details in new ticket spun up.

**Comment by jhouse-solvd** [comment]
@oseasmoran73 & @zachclarity - I think it could be good to copy these reviews to the SRE Confluence space so that we can check off things as they are addressed. I think we could link to this issue and others as needed. Thoughts? 

I'd feel better closing this out once the reviews are documented.

**Comment by zachclarity** [comment]
# Issue Closed with the following:

## The following issues tasks are open for this review.  

- [x] [Review incident commander documentation](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41148)
- [x] [Review incident categorization documentation](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41151)
- [x] [Review incident response playbook](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41153)
